unitOneAssessmentPractice.js

```diff
@@ -3,39 +3,100 @@ let assert = require('assert')
 // Question One:
```

Comment:
Overall good performance for this review. The only problem that you didn't solve yet is Question 6 which is tough question to crack due to using object. Let me know if you have any questions regarding that problem.